samples/nrf_rpc/entropy_nrf53/cpuapp/prj.conf

```diff
@@ -9,10 +9,12 @@ CONFIG_HEAP_MEM_POOL_SIZE=4096
 
 CONFIG_OPENAMP=y
```

Comment:
do you need to enable Open AMP now or does it automatically is enabled when enabling the RPMsg service:

CONFIG_RPMSG_SERVICE=y

Reply:
Yes, it is not necessary to enable Open AMP directly.